### PR TITLE
make routes cacheable (close #344)

### LINKED ISSUE #344: Change how routes are loaded
currently they are loaded in runtime avoiding caches
like this
```php
$router->group($config, function($router)
        {
            $router->get('view/{groupKey?}', 'Controller@getView')->where('groupKey', '.*');
            $router->get('/{groupKey?}', 'Controller@getIndex')->where('groupKey', '.*');
            $router->post('/add/{groupKey}', 'Controller@postAdd')->where('groupKey', '.*');
            $router->post('/edit/{groupKey}', 'Controller@postEdit')->where('groupKey', '.*');
            $router->post('/groups/add', 'Controller@postAddGroup');
            $router->post('/delete/{groupKey}/{translationKey}', 'Controller@postDelete')->where('groupKey', '.*');
            $router->post('/import', 'Controller@postImport');
            $router->post('/find', 'Controller@postFind');
            $router->post('/locales/add', 'Controller@postAddLocale');
            $router->post('/locales/remove', 'Controller@postRemoveLocale');
            $router->post('/publish/{groupKey}', 'Controller@postPublish')->where('groupKey', '.*');
        });
```
what is recomended in docs is https://laravel.com/docs/6.x/packages#routes
```php
    $this->loadRoutesFrom(__DIR__.'/routes.php');
```

### PR DESCRIPTION
**Pull request by vladyslavstartsev**
Made changes so that we can cache routes (close #344)

### CODE REVIEW

**Comment by barryvdh** [comment]
But now the config is gone?

**Comment by vladyslavstartsev** [comment]
@barryvdh can you clarify please? 

I don't really understand.

I haven't touched config files

**Comment by vladyslavstartsev** [comment]
oh, I got it, sorry will fix with working config

File: src/routes.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+$config = array_merge(config('translation-manager.route'), ['namespace' => 'Barryvdh\TranslationManager']);
```
Comment (vladyslavstartsev) [inline src/routes.php]: I made it work through this.

It looks a bit ugly since we are not configuring in ServiceProvider anymore

**Comment by vladyslavstartsev** [comment]
is there a way to merge this?

**Pull request merged**